docs/VnVPlan/VnVPlan.tex

```diff
@@ -224,44 +225,152 @@ \subsection{Tests for Functional Requirements}
 \wss{Include a blurb here to explain why the subsections below
 	cover the requirements.  References to the SRS would be good here.}
 
-\subsubsection{Area of Testing1}
-
-\wss{It would be nice to have a blurb here to explain why the subsections below
-	cover the requirements.  References to the SRS would be good here.  If a section
-	covers tests for input constraints, you should reference the data constraints
-	table in the SRS.}
+\subsubsection{Appointments}
 
-\paragraph{Title for Test}
+This section will contains tests covering the ``Appointments'' requirements defined in the
+\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}.
 
 \begin{enumerate}
 
-	\item{test-id1\\}
+	\item{FRT-BE4-1}
 
-	Control: Manual versus Automatic
+	Control: Manual
 
-	Initial State:
+	Initial State: Customer account got a quote from an auto shop
 
-	Input:
+	Input: Enters service request information and selects an available appointment time slot
 
-	Output: \wss{The expected result for the given inputs}
+	Output: A new appointment is created with service details
```

Comment:
```suggestion
	Output: A new appointment is created with service details linked to the quote
```

Reply:
Quote should also show the appointment was made right?

Reply:
Should the shop get the request as output? Does the shop need to accept the appointment? If so, this is an appointment request right? Same for below